integration-tests/README.md

```diff
@@ -13,7 +13,7 @@ sudo npm install -g yarn
    for more information:
 
 ```bash
-cargo run --release -- --dev --mining-key XYZ
+cargo run --release -- --dev --mining-key 5DkPYq8hFiCeGxFBkz6DAwnTrvKevAJfTYrzFtr9hpDsEAU1 --monitor-nonce auto
```

Comment:
Can you elaborate?

Reply:
A key used in multiple places during testing. I've changed the instructions so that you can copy & paste the command without needing to do anything else.